### PR TITLE
Add Press Page to Celo.org

### LINKED ISSUE #4662: Add Press page to celo.org
This will be a place to list when celo is mentioned in press. 

Something similar to the layout of /papers

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Add a Press page which lists all the times Celo has been mentioned in the press. 

### Tested

Screen Shot 2020-08-18 at 12 23 01 PM

### Related issues

- Fixes #4662 

### Backwards compatibility

new

### CODE REVIEW

File: packages/web/src/press/PressPage.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  press: PressArticleFields[]
+}
+
+class PressPage extends React.PureComponent<I18nProps & Props> {
+  static async getInitialProps(context) {
+    let press = []
+    try {
+      if (context.req) {
+        const getpress = await import('src/../server/fetchPress')
+        press = await getpress.default()
```
Comment (jmrossy) [inline packages/web/src/press/PressPage.tsx:32]: Can you clarify what's going on here?

Comment (aaronmgdr) [reply packages/web/src/press/PressPage.tsx:32]: added a comment. 

on server import module and call code directly. on client call the https endpoint

Comment (aaronmgdr) [reply packages/web/src/press/PressPage.tsx:32]: import dynamically so code isn't imported on browser.

File: packages/web/src/_page-tests/experience/__snapshots__/exchange-icons.test.tsx.snap [hunk 2]
```diff
@@ -4230,6 +4230,59 @@ exports[`Experience/ExchangeIcons renders 1`] = `
...
+                      "lineHeight": "20px",
+                      "opacity": 1,
+                      "textDecoration": "none",
+                      "textDecorationStyle": "solid",
+                      "textRendering": "geometricPrecision",
+                    }
+                  }
+                  target="_blank"
+                >
+                  Presss
```
Comment (ErinHales) [inline packages/web/src/_page-tests/experience/__snapshots__/exchange-icons.test.tsx.snap]: Extra 's' :)

File: packages/web/src/shared/menu-items.ts [hunk 3]
```diff
@@ -71,6 +71,10 @@ export const pagePaths = {
     name: 'White Papers',
     link: '/papers',
   },
+  PRESS: {
+    name: 'Presss',
```
Comment (ErinHales) [inline packages/web/src/shared/menu-items.ts]: It looks like this is where that extra 's' is coming from in the other files

**Review by ErinHales — CHANGES_REQUESTED** [review]
Typo, but other than that it looks good

File: packages/web/src/shared/menu-items.ts (hunk 3 above)
Comment (aaronmgdr) [reply packages/web/src/shared/menu-items.ts]: ah good call.

Comment (aaronmgdr) [reply packages/web/src/shared/menu-items.ts]: all fixed

**Review by ErinHales — APPROVED** [review]

**Pull request merged**